templates/acs-deployment-master.yaml

```diff
@@ -438,10 +450,26 @@ Parameters:
       Description: The private registry credentials in base64 format.
       NoEcho: True
       Default: ""
+    RepoImage:
+      Type: String
+      Description: The Alfresco Repository docker image name
+      Default: "alfresco/alfresco-content-repository-aws"
+    RepoTag:
+      Type: String
+      Description: The Alfresco Repository docker tag name
+      Default: "6.1.0-RC7"
     RepoPods:
       Type: String
       Description: The number of repository pods in the cluster
       Default: "2"
+    ShareImage:
+      Type: String
+      Description: The Alfresco Share docker image name
+      Default: "alfresco/alfresco-share"
```

Comment:
Here too Default: "" Maybe we could mention in the description that it will be from docker hub alfresco/alfresco-share  . Would be nice to know for the developer.